ocaml/testsuite/tests/typing-layouts/error_message_attr.ml

```diff
@@ -50,15 +90,16 @@ Line 3, characters 61-63:
                                                                  ^^
 Error: This type ('a : value) should be an instance of type ('b : float64)
        The layout of 'a is value, because
-         the [@error_message] attribute: Custom message.
+         of the annotation on the type variable 'a
+         Custom message.
```

Comment:
People tend to read the beginning and end of error messages. Putting the custom message in the middle seems like it might get lost. But I think fixing this would be hard. If you agree, then let's not for now. But maybe you see an easy way of pulling this out.

Reply:
I agree it does seem hard to fix this. Will have to add something at every `format_history` call site (two places at the moment) to check for `With_error_message`.